### DISCUSSION TITLE
get_schema_view() OpenAPI fixed fields

### DISCUSSION THREAD

**Discussion by fgiudici95** [encode/django-rest-framework#7996] [Ideas & Suggestions]
Hi,
Reading the documentation and analyzing the code I noticed that it is not possible to generate a OpenAPI file using the `get_schema_view()` function a complete [Info Object](https://swagger.io/specification/#info-object).  
I think that would be very interesting to just insert the [Contact Object](https://swagger.io/specification/#contact-object) and the [License Object](https://swagger.io/specification/#license-object)

What do you think?
Cheers
Fabio

**Comment by n2ygk** [comment]
You can extend get_schema() and add anything you want:
```python
    def get_schema(self, request, public):
        schema = super().get_schema(request, public)
        schema['info'] = {
            'version': __version__,
            'title': __title__,
            'description':
         ...
        return schema
```

**Comment by n2ygk** [comment]
The documented example is here:
https://www.django-rest-framework.org/api-guide/schemas/#get_schemaself-request

**Reply by fgiudici95** [reply]
Thank you!
I missed it. 
I'll close the discussion!
